test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpLogExporterTests.cs

```diff
@@ -44,32 +42,215 @@ public void ToOtlpLogRecordTest()
+                builder.AddOpenTelemetry(options =>
+                {
+                    options.IncludeFormattedMessage = true;
+                    options.ParseStateValues = true;
+                    options.AddInMemoryExporter(logRecords);
+                });
+            });
+
+            var logger = loggerFactory.CreateLogger("OtlpLogExporterTests");
             logger.LogInformation(new EventId(10, null), "Hello from {name} {price}.", "tomato", 2.99);
             Assert.Single(logRecords);
-            logRecord = logRecords[0];
-            otlpLogRecord = logRecord.ToOtlpLog();
+
+            var logRecord = logRecords[0];
+            var otlpLogRecord = logRecord.ToOtlpLog();
+
             Assert.NotNull(otlpLogRecord);
             Assert.Equal("Hello from tomato 2.99.", otlpLogRecord.Body.StringValue);
+
+            var otlpLogRecordAttributes = otlpLogRecord.Attributes.ToString();
+
+            // Event
+            Assert.Contains("Id", otlpLogRecordAttributes);
+            Assert.Contains("10", otlpLogRecordAttributes);
+
             logRecords.Clear();
 
             logger.LogInformation(new EventId(10, "MyEvent10"), "Hello from {name} {price}.", "tomato", 2.99);
             Assert.Single(logRecords);
+
             logRecord = logRecords[0];
             otlpLogRecord = logRecord.ToOtlpLog();
             Assert.NotNull(otlpLogRecord);
             Assert.Equal("Hello from tomato 2.99.", otlpLogRecord.Body.StringValue);
+
+            otlpLogRecordAttributes = otlpLogRecord.Attributes.ToString();
+
+            // Event
+            Assert.Contains("Id", otlpLogRecordAttributes);
+            Assert.Contains("10", otlpLogRecordAttributes);
+            Assert.Contains("Name", otlpLogRecordAttributes);
+            Assert.Contains("MyEvent10", otlpLogRecordAttributes);
+        }
+
+        [Fact]
+        public void CheckToOtlpLogRecordTraceIdSpanIdFlagWithDroppedActivity()
+        {
+            List<LogRecord> logRecords = new List<LogRecord>();
+            using var loggerFactory = LoggerFactory.Create(builder =>
+            {
+                builder.AddOpenTelemetry(options =>
+                {
+                    options.AddInMemoryExporter(logRecords);
+                });
+            });
+
+            var logger = loggerFactory.CreateLogger("OtlpLogExporterTests");
+            logger.LogInformation("Log within a dropped activity");
+            var logRecord = logRecords[0];
+            var otlpLogRecord = logRecord.ToOtlpLog();
+
+            Assert.Null(Activity.Current);
+            Assert.True(otlpLogRecord.TraceId.IsEmpty);
+            Assert.True(otlpLogRecord.SpanId.IsEmpty);
+            Assert.True(otlpLogRecord.Flags == 0);
+        }
+
+        [Fact]
+        public void CheckToOtlpLogRecordSpanIdTraceIdAndFlag()
+        {
+            // preparation
+            var exportedActivityList = new List<Activity>();
+            var activitySourceName = "toOtlpLogRecordTest";
```

Comment:
Consider using something like this so we don't have to worry about name conflicts:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/1b5765bf43058c6301e0ff0c099aedb34541d54e/test/OpenTelemetry.Tests/Metrics/MetricAPITest.cs#L46